db/migrate/20181105190457_create_votes.rb

```diff
@@ -2,9 +2,9 @@ class CreateVotes < ActiveRecord::Migration[5.2]
   def change
     create_table :votes do |t|
       t.references :author, foreign_key: true
-      t.references :post, foreign_key: true
-
+      t.references :voting_object, index: true, polymorphic: true
       t.timestamps
     end
+    add_foreign_key :votes, :votables
```

Comment:
nie ma tabeli `votables` i dlatego travis się wysypuje

Reply:
Nie moża też zmieniać istniejących migracji. One już się wykonały i drugi raz nie zostaną wykonane. Należy cofnąć te zmiany.

Zamiast tego trzeba dodawać nowe migracje tak jak zostało to zrobione np. w db/migrate/20181126204157_add_voting_object_id_to_votes.rb